application/backend/src/main/resources/apps/typerefinery/components/structure/page/clientlibs-header/style.css

```diff
@@ -10,4 +10,8 @@ body {
 
 .bg-secondary {
     background-color: #C7DEFA !important;
+}
+
+.row {
```

Comment:
can we add a comment to this so we know why its removed?

"remove global row gutter to ensure all children components do not have padding added, this is replaced by individual components specifying their own padding."